lib/packages/docutils/rstgen.nim

```diff
@@ -675,7 +675,7 @@ proc readIndexDir(dir: string):
           var x = fileEntries[0].link
           let i = find(x, '#')
           if i > 0:
-            x = x.substr(0, i-1)
+            x.setLen(i-1)
```

Comment:
obviously, the range of `i` is 0 .. x.len-1

Reply:
shouldn't this be:

```nim
x.setLen i
```

?

Reply:
Yeah, I realized it when I went to sleep yesterday.